teach/static/admin/js/toggle_clubs_guides_translation.js

```diff
@@ -0,0 +1,27 @@
+(function() {
+    var $ = django.jQuery.bind(django.jQuery);
+
+    function allowChoosingTranslation() {
+        $(".field-category").hide();
+        $(".field-translation_of").show();
+    }
+
+    function disallowChoosingTranslation() {
+        $(".field-category").show();
+        $(".field-translation_of").hide();
+    }
+
+    document.addEventListener("DOMContentLoaded", function() {
+        var isTranslationYes = $("#id_is_translation_0");
+        var isTranslationNo = $("#id_is_translation_1");
+
+        if(isTranslationYes.is(":checked")) {
```

Comment:
this should really be done in the toggle function: when triggered, check which state to show, and then show that. Also, does this initial check make sense? Can the "new club" ever start with "is translation" already checked? Because https://github.com/mozilla/teach-api/pull/64/files#diff-72ae060e51d3a2819400bd8ab56113adR15 suggests that is impossible...